src/Console/Application.php

```diff
@@ -53,18 +53,16 @@
 final class Application extends BaseApplication
 {
     private const NAME = 'Infection - PHP Mutation Testing Framework';
-    private const VERSION = '@package_version@';
 
     private const INFECTION_PREFIX = 'INFECTION';
 
-    private const LOGO = <<<'ASCII'
-     ____      ____          __  _
-    /  _/___  / __/__  _____/ /_(_)___  ____
-    / // __ \/ /_/ _ \/ ___/ __/ / __ \/ __ \
-  _/ // / / / __/  __/ /__/ /_/ / /_/ / / / /
- /___/_/ /_/_/  \___/\___/\__/_/\____/_/ /_/
-
-ASCII;
```

Comment:
I would suggest to revert adding 4 space instead of 1, it looks strange https://travis-ci.org/infection/infection/jobs/599743124#L812

Reply:
can we make it aligned the same way as composer?

<img width="647" alt="composer_align" src="https://user-images.githubusercontent.com/3725595/67979359-22422980-fc2d-11e9-9362-f214818efa4f.png">